mqttx-web/umbrel-app.yml

```diff
@@ -0,0 +1,22 @@
+manifestVersion: 1
+id: mqttx-web
+name: MQTTX Web
+tagline: Your All-in-One MQTT Client Toolbox
+icon: ""
+category: automation
+version: "1.11.0"
+port: 1884
+description: >-
+  MQTTX Web is an open-source MQTT 5.0 WebSocket client on the browser to help you debug your MQTT services and applications faster.
+
+  The user interface (UI) of MQTTX adopts a chat-based layout, simplifying operational logic. It enables users to establish multiple M>
+developer: EMQ
+website: https://mqttx.app/
+submitter: dirstel
+submission: https://github.com/getumbrel/umbrel-apps/pull/1772
+repo: https://github.com/emqx/MQTTX
+support: https://github.com/emqx/MQTTX
```

Comment:
It looks like their website links here for support:
`https://github.com/emqx/MQTTX/discussions`

Reply:
fixed: changed support url